lib/tailwindcss/commands.rb

```diff
@@ -72,6 +72,23 @@ def watch_command(poll: false, **kwargs)
           command << "-p" if poll
         end
       end
+
+      def debug_option(rake_task_args_extras)
+        rake_task_args_extras.include?("debug") || truthy_string_value(ENV['TAILWINDCSS_RAILS_NO_MINIFY'])
+      end
+
+      def poll_option(rake_task_args_extras)
+        rake_task_args_extras.include?("poll")
+      end
+
+      def truthy_string_value(value)
+        if !value.present? ||
```

Comment:
Do we have `present?` available? The core_ext is not being required in this gem. I think it only works in tests because the app is configured to require all core_ext.